File: forge/src/main/java/dev/gluton/quickspyglasser/forge/QuickSpyglasserExpectPlatformImpl.java
package dev.gluton.quickspyglasser.forge;

import dev.gluton.quickspyglasser.QuickSpyglasserExpectPlatform;
import net.minecraftforge.fml.loading.FMLPaths;

import java.nio.file.Path;

public class QuickSpyglasserExpectPlatformImpl {
    /**
     * This is our actual method to {@link QuickSpyglasserExpectPlatform#getConfigDirectory()}.
     */
    public static Path getConfigDirectory() {
        return FMLPaths.CONFIGDIR.get();
    }
}

File: forge/src/main/kotlin/dev/gluton/quickspyglasser/forge/QuickSpyglasserExpectPlatformImpl.kt
package dev.gluton.quickspyglasser.forge

import net.minecraftforge.fml.loading.FMLPaths
import java.nio.file.Path

object QuickSpyglasserExpectPlatformImpl {
	@JvmStatic
	fun configDirectory(): Path = FMLPaths.CONFIGDIR.get()
}
